nbbuild/cluster.properties

```diff
@@ -248,6 +248,8 @@ nb.cluster.platform=\
         net.java.html.json,\
         net.java.html.sound,\
         netbinox,\
+        o.apache.commons.io,\
```

Comment:
I don't think we should add new unavoidable libraries to the `platform` cluster.

Reply:
Marking as resolved since JARs end up in platform/modules rather than platform/lib, per the last few comments.